config/install/core.entity_form_display.node.page.default.yml

```diff
@@ -2,75 +2,91 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.field.node.page.body
+    - field.field.node.page.field_body
```

Comment:
Let's have one field called "Paragraphs" with unlimited cardinality.
This field will have two available options for now: "Body" and "FAQ".